sysfs/class_fibrechannel_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"k8s.io/utils/ptr"
```

Comment:
While this is a convenient package, I'm not sure we want to add this to the dependency tree.

This package is pulled in by a lot of places and I would like to minimize the indirect requirements.

In other similar tests, it's painful, but we do string vars and then `&stringVar`.